mobile/lib/resources/text_styles.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/cupertino.dart';
+import 'package:radioaktywne/extensions/build_context.dart';
+
+class RATextStyles {
+  const RATextStyles(this.context);
+
+  final BuildContext context;
+
+  TextStyle get polibudzka => TextStyle(
+        fontSize: 18,
+        fontFamily: 'RadikalWUT',
+        fontWeight: FontWeight.bold,
+        color: context.colors.highlightGreen,
+      );
+
+  TextStyle get textPlayer => TextStyle(
+        fontSize: 24,
+        fontFamily: 'Arial',
+        fontWeight: FontWeight.bold,
+        color: context.colors.highlightGreen,
+      );
+
+  TextStyle get textMedium => TextStyle(
+        fontSize: 16,
+        fontFamily: 'Arial',
+        fontWeight: FontWeight.bold,
+        color: context.colors.highlightGreen,
+      );
+
+  TextStyle get textSmall => TextStyle(
+        fontSize: 12,
+        fontFamily: 'Arial',
+        fontWeight: FontWeight.bold,
+        color: context.colors.highlightGreen,
+      );
+}
```

Comment:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Honestly, I don't know why GitHub doesn't show newline character at the end of the file, but it is there in my code editor (and in the file itself, too).